src/helpers/dom/element.js

```diff
@@ -762,11 +762,55 @@ export function getComputedStyle(element, rootWindow = window) {
 /**
  * Returns the element's outer width.
  *
+ * @see {preciseOuterWidth}
  * @param {HTMLElement} element An element to get the width from.
  * @returns {number} Element's outer width.
  */
 export function outerWidth(element) {
-  return Math.ceil(element.getBoundingClientRect().width);
+  return element.offsetWidth;
+}
+
+/**
+ * Calculates the current `transform` `scaleX` property value for the passed in
+ * element, relative to the root document. Involves adding a dummy element of a
+ * constant size to the DOM, be cautious about performance.
+ *
+ * @param {HTMLElement} element An element to get the `scaleX` from.
+ * @returns {number} `scaleX` value.
+ */
+export function computeScaleX(element) {
+  // A dummy element with a constant size, used to calculate the `scale`.
+  const $dummy = element.ownerDocument.createElement('div');
+  const factor = 100;
+
+  $dummy.style.width = `${factor}px`;
+  $dummy.style.display = 'inline-block';
+  $dummy.style.position = 'absolute';
+  $dummy.style.left = '0';
+  $dummy.style.top = '0';
+  $dummy.style.visibility = 'hidden';
+
+  element.appendChild($dummy);
+
+  const dummyRect = $dummy.getBoundingClientRect();
+  const scaleX = dummyRect.width / factor;
+
+  $dummy.remove();
```

Comment:
The [remove()](https://developer.mozilla.org/pl/docs/Web/API/ChildNode/remove) method is not supported by IE 🤯  The Handsontable supports IE. Can we change that?

Reply:
Good catch, a fix was quite preferable as without it the whole table would break on IE, even on IE11. https://github.com/handsontable/handsontable/pull/7496/commits/1bc9df6cf55f488c043f1d5b3417ff91e6ff7139